File: go_service/common/utils/constant.go
package utils

const (
	maxFileInfoNum    = 1024                   // 一个文件夹下最多的文件信息个数
	yaml              = "yaml"                 // 配置文件类型
	fileNameRegex     = "^[\\s\\S]+.[a-zA-z]$" // 文件名称的正则表达式
	filePathRegex     = ""
	ModuleApplication = "application" // 应用的基础设置
	ModuleMySQL       = "mysql"
	ModuleRedis       = "redis"

	// 通用的 error_code 如果不指明错误类型
	// 则默认是这个类型的code
	CommonErrorCode = 400
	NotFoundError   = 404
	ServerError     = 500
	SuccessCode     = 200

	Second = 1
	Minute = 60 * Second

	ValidatorKey   = "ValidatorKey"
	TranslatorKey  = "TranslatorKey"
	SessionKeyUser = "user"

	NeedHttps   = 1
	NoNeedHttps = 0

	NumBase10 = 10
	NumBase2  = 2

	IntegerBitSize64 = 64
	IntegerBitSize32 = 32

	HTTPLoadType = 0
	TCPLoadType  = 1
	GrpcLoadType = 2

	HTTPRuleTypePrefixURL = 0
	HTTPRuleTypeDomain    = 1

	HttpServiceType = "http"
	TcpServiceType  = "tcp"
	GrpcServiceType = "grpc"
	AllServiceType  = "all"
)

var ERROR_CODE = []int{
	CommonErrorCode,
	NotFoundError,
	ServerError,
}

File: java_service/src/main/java/com/kyriexu/utils/Constant.java
package com.kyriexu.utils;

/**
 * @author KyrieXu
 * @since 2021/3/23 12:49
 **/
public class Constant {
    public final static int SUCCESS = 200;
    public final static int COMMON_FAIL = 400;

    public final static int SIZE_LIMIT = 50;

    public final static String SERVICE_ALL = "all";
    public final static String SERVICE_TCP = "tcp";
    public final static String SERVICE_HTTP = "http";
    public final static String SERVICE_GRPC = "grpc";

    public final static int HTTPLoadType = 0;
    public final static int TCPLoadType = 1;
    public final static int GrpcLoadType = 2;


    public final static int HTTPRuleTypePrefixURL = 0;
    public final static int HTTPRuleTypeDomain = 1;

    public final static int NeedHttps = 1;
    public final static int NoNeedHttps = 0;


    public final static String[] SERVICE_TYPE = new String[]{
            SERVICE_ALL,
            SERVICE_TCP,
            SERVICE_HTTP,
            SERVICE_GRPC
    };


    public final static String USER = "user";
}
